# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/src/frontend/Stencil.cpp
# Commit: b1796f9dbc1e
# Full Hash: b1796f9dbc1e490b29f81cb9b674f0534cc3f44a
# Author: Tooru Fujisawa <arai_a@mac.com>
# Date: 2020-11-17 09:44:06
# Description:
#   Bug 1677371 - Add missing return. r=tcampbell
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D97193
# ==============================================================================

diff -r dc0e5a789feb -r b1796f9dbc1e js/src/frontend/Stencil.cpp
--- a/js/src/frontend/Stencil.cpp	Mon Nov 16 21:06:12 2020 +0200
+++ b/js/src/frontend/Stencil.cpp	Mon Nov 16 17:37:34 2020 +0000
@@ -999,6 +999,7 @@
     GenericPrinter& out = json.beginStringProperty("atom");
     WellKnownParserAtoms::getStatic2(index)->dumpCharsNoQuote(out);
     json.endString();
+    return;
   }
 
   MOZ_ASSERT(taggedIndex.isNull());